tests/FakeItEasy.Specs/CreationOptionsSpecs.cs

```diff
@@ -6,12 +6,12 @@ namespace FakeItEasy.Specs
     using System.Diagnostics.CodeAnalysis;
     using System.Linq;
     using System.Reflection.Emit;
+    using Creation;

```

Comment:
I see we already have mix here, but I think we usually prefer fully qualified namespaces.